src/components/application_manager/test/commands/mobile/system_request_test.cc

```diff
@@ -128,6 +140,133 @@ TEST_F(SystemRequestTest, Run_HTTP_FileName_no_binary_data_REJECTED) {
   command->Run();
 }
 
+TEST_F(SystemRequestTest,
+       Run_RequestTypeAllowedAndRequestSubTypeAllowed_SendHMIRequest) {
+  MessageSharedPtr msg = CreateIVSUMessage();
```

Comment:
@BSolonenko Same as above

Reply:
@ValeriiMalkov I don't see in this any sense

Reply:
@BSolonenko Please, use C++11 https://adc.luxoft.com/confluence/pages/viewpage.action?pageId=307580753